src/main/java/com/direwolf20/buildinggadgets/client/gui/ModeRadialMenu.java

```diff
@@ -175,7 +184,7 @@ public void drawScreen(int mx, int my, float partialTicks) {
             if (ysp < y)
                 ysp -= 9;
 
-            fontRenderer.drawStringWithShadow(name, xsp, ysp, 0xFFFFFF);
+            fontRenderer.drawStringWithShadow(name, xsp, ysp, i == modeIndex ? Color.GREEN.getRGB() : 0xFFFFFF);
```

Comment:
I don't know whether it would be better to read if we did:
```suggestion
            fontRenderer.drawStringWithShadow(name, xsp, ysp, i == modeIndex ? Color.GREEN.getRGB() :  Color.WHITE.getRGB());
```